lib/ansible/modules/cloud/azure/azure_rm_containerinstance.py

```diff
@@ -0,0 +1,429 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2017 Zim Kalinowski, <zikalino@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_containerinstance
+version_added: "2.5"
+short_description: Manage an Azure Container Instance.
+description:
+    - Create, update and delete an Azure Container Instance.
+
+options:
+    resource_group:
+        description:
+            - Name of resource group.
+        required: true
+    name:
+        description:
+            - The name of the container group.
+        required: true
+    os_type:
+        description:
+            - The OS type of containers.
+        choices:
+            - linux
+            - windows
+        default: linux
+    state:
+        description:
+            - Assert the state of the container instance. Use 'present' to create or update an container instance and 'absent' to delete it.
+        default: present
+        choices:
+            - absent
+            - present
+    ip_address:
+        description:
+            - The IP address type of the container group (default is 'none')
+        choices:
+            - public
+            - none
+        default: None
+    ports:
+        description:
+            - List of ports exposed within the container group.
+    location:
+        description:
+            - Valid azure location. Defaults to location of the resource group.
+        default: resource_group location
```

Comment:
Unless it's a specific value, leave the default out and put it in the description like you have done.